client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/navigator-item-group/style.scss

```diff
@@ -0,0 +1,12 @@
+@import "@automattic/typography/styles/fonts";
+
+.pattern-assembler {
+	.pattern-layout__label {
+		color: var(--color-neutral-100);
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg)

I used `--color-neutral-100` here since we're using `--color-neutral-100` for `Let’s get creative` and an item label (e.g. `Header`, `Footer`).
<img width="321" alt="Screen Shot 2023-04-05 at 15 30 09" src="https://user-images.githubusercontent.com/5287479/229999067-94cc7ad3-f010-4094-9e8d-3e303aa6bda0.png">

If this makes sense, should we update Figma that way as well? `--color-neutral-100` is `Gray 100` on Figma. @davewhitley 
<img width="464" alt="Screen Shot 2023-04-05 at 15 33 07" src="https://user-images.githubusercontent.com/5287479/229999640-1889c306-ab02-4d33-ba49-0dec509a5943.png">

Reply:
> It's not feasible for the Figma mockups and live code to be perfectly in sync

I agree with it. 
I tried something like ”Code generation from design” or "[Connecting it to Storybook (the code)](https://www.figma.com/community/plugin/1056265616080331589/Storybook-Connect)", but syncing the design file and the code was always challenging.

Glad to have that confirmed. Thank you! @davewhitley